lib/modules/platform/bitbucket/index.spec.ts

```diff
@@ -3,9 +3,9 @@ import type { logger as _logger } from '../../../logger';
 import { BranchStatus, PrState } from '../../../types';
 import type * as _git from '../../../util/git';
 import { setBaseUrl } from '../../../util/http/bitbucket';
-import type { Platform, RepoParams } from '../types';
+import type { Platform, PlatformResult, RepoParams } from '../types';
 
-const baseUrl = 'https://api.bitbucket.org';
+const baseUrl = 'https://api.bitbucket.org/';
```

Comment:
Not sure exactly why this URL lacks the trailing `/` when the other places it's defined (like [here](https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/bitbucket/index.ts#L43) and [here](https://github.com/renovatebot/renovate/blob/main/lib/util/http/bitbucket.ts#L5) it includes the trailing `/`

Reply:
revert, we shoud generally don't use trailing slashes. most of the time we trim them anyways